config/locales/members.en.yml

```diff
@@ -94,8 +94,24 @@ en:
       pictures: Pictures
       vacancies: Vacancies
     payments:
+      mongoose:
+        credit: Credit
+        subtitle: You can solely use mongoose credit to pay for product in Sticky's room.
```

Comment:
You can **only** use **M**ongoose credit to pay for product**s** in Sticky's room.